src/components/Reservations.jsx/Reservations.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react'
```

Comment:
- [ ] Is not directly related to this PR, but always keep in mind that any user logged or not could directly type the URL and access this page. Please create an issue to solve this problem.

Reply:
I know but I wanted to discuss with you guys if we are going to agree on a more general approach to this or if I just make another protected route?

Reply:
Let's have a meeting about this concern after lunch break because I know a way to do it, but I'm not sure if that's the best way to do it also. Anyways, please feel free to create an issue if you feel it's necessary